ubercj/hacktoberfest-tracking
# Change shirtSize from text field to select dropdown

closes #11 
# Description #
Change shirtSize from text field to select dropdown.  Example we want the user to have the ability to choose from a predefined shirt sizes dropdown.  Also when the user has not chosen a shirt size yet or has just created their user account the dropdown will say 'Select a Size'.  This will work with the data that already exist as it has been saved in the database as a string.

- Created array of shirt sizes
- defaulting `shirtSize` as an empty string
- Created select with options
- Create class to style dropdown exactly like text inputs

# Testing #

1. Login and go to Dashboard
2. Validate that your shirt size is selected, if no shirt size has been selected validate 'Select a Size' is shown
3. Select a shirt size different than what already is selected
4. Press 'UPDATE PROFILE' button
5. Refresh page after saving
6. Validate new selected shirt size is selected in dropdown and next to 'Shirt Size:' in the Dashboard

# Screenshots #
No Shirt Size Selected
![Screen Shot 2022-10-17 at 6 44 39 PM](https://user-images.githubusercontent.com/14244920/196298501-42d1ef47-2984-4cc4-a6c8-f5d3d2d8e100.png)

Selecting Shirt Size
![Screen Shot 2022-10-17 at 6 44 44 PM](https://user-images.githubusercontent.com/14244920/196298506-a571db61-4a0c-4809-8732-a7c8fa3ef1f6.png)

Shirt Size Selected
![Screen Shot 2022-10-17 at 6 54 09 PM](https://user-images.githubusercontent.com/14244920/196298569-b066e5b4-fe37-4ad9-92ab-ceac153a8425.png)
